ARCHITECTURE.md

```diff
@@ -48,7 +48,7 @@ export default class AppCard extends Vue {
 <style lang="scss" scoped>
 @import "~carbon-components/scss/globals/scss/typography";
 
-.app-card {
+.card {
```

Comment:
above in [Line 42](https://github.com/Qiskit/qiskit.org/pull/3073/files#diff-8f6366fd8e7a5fa30f7f05879999195b7cf5fa1e3d157822eb37f0e91d226cfdR42), should we also update this?

Reply:
Fixed in https://github.com/Qiskit/qiskit.org/pull/3073/commits/0f01cdbea3f938f91cf1cfdf1fe185f667ce7dc6